app/pods/components/agenda/agenda-detail/sidebar-item/component.js

```diff
@@ -53,7 +53,7 @@ export default class SidebarItem extends Component {
 
   @action
   conditionallyScrollIntoView(element) {
-    if (this.isActive) {
+    if (!this.isActive) {
```

Comment:
Er zal wel een reden zijn waarom je dit doet, maar voorheen werkte dit prima ... ik vermoed dat er elders iets verschoven is (en elders terug op z'n plaats moet gezet worden) :thinking: ?

Reply:
Yes, nu werkt het omgekeerd, heel raar waarom dit ineens nu zo zou moeten.. nochtans heb ik het hertest en gaat nu alles zoals het zou moeten werken (lijkt me).